vendor/k8s.io/kubernetes/openshift-kube-apiserver/admission/network/apis/externalipranger/v1/register.go

```diff
@@ -1,17 +1,19 @@
 package v1
 
 import (
-	"k8s.io/kubernetes/openshift-kube-apiserver/admission/network/apis/restrictedendpoints"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/kubernetes/openshift-kube-apiserver/admission/network/apis/externalipranger"
 )
 
+func (obj *ExternalIPRangerAdmissionConfig) GetObjectKind() schema.ObjectKind { return &obj.TypeMeta }
+
 var GroupVersion = schema.GroupVersion{Group: "network.openshift.io", Version: "v1"}
 
 var (
 	schemeBuilder = runtime.NewSchemeBuilder(
 		addKnownTypes,
-		restrictedendpoints.Install,
+		externalipranger.Install,
```

Comment:
hmmm... how was this not caught before?

Reply:
🤷‍♂ . I guess it's just a typo. The install method forwards on to `schemeBuilder.AddToScheme` in both cases, so it probably doesn't matter.